CHANGELOG.md

```diff
@@ -0,0 +1,40 @@
+- [server] The `org.apache.log4j` dependency is not reexported any more when consumed as p2 artifact. You need to import `org.apache.log4j` in your own plugins if you use the log4j API there. [#430](https://github.com/eclipse-glsp/glsp/issues/430)
+- [di] Complete rework of DI architecture. Move away from one global injector per application. Instead use one global server injector and dedicated child injectors for each diagram session. [#150](https://github.com/eclipse-glsp/glsp-server/pull/127)
+- [launch] Improve customizability of `DefaultGLSPServerLauncher`. [#385](https://github.com/eclipse-glsp/glsp-server/pull/128)
+- [model] Add properties map in model state to store arbitrary properties that are persistent between handler calls. [#401](https://github.com/eclipse-glsp/glsp-server/pull/132)
+- [server] Refactor json-rpc protocol and `DefaultGLSPServerImplementation`. [#421](https://github.com/eclipse-glsp/glsp-server/pull/133)
+- [action] Rework `ActionHandler` & `OperationHandler` API. [#425](https://github.com/eclipse-glsp/glsp-server/pull/135)
+- [action] Introduce `dispatchAfterNextUpdate` functionality for the `ActionDispatcher`. This enables queuing of actions until the next model updates has been processed. [#448](https://github.com/eclipse-glsp/glsp-server/pull/141/)
+- [protocol] Align server-side action definitions with [protocol definition](https://github.com/eclipse-glsp/glsp/blob/master/PROTOCOL.md). [#432](https://github.com/eclipse-glsp/glsp-server/pull/142)
+
+[Breaking Changes:](#breaking_changes_1.9.0)
+
+- [diagram] Merged the `ServerLayoutConfiguration` API into the `DiagramConfiguration` API. The standalone `ServerLayoutConfiguration` is now deprecated and no longer supported [#123](https://github.com/eclipse-glsp/glsp-server/pull/95)
+- [model] Renamed `ModelFactory` to `ModelSourceLoader` and adapted interface method. This also affects implementing classes like the `JsonFileModelFactory` [#119](https://github.com/eclipse-glsp/glsp-server/pull/96)
+- [model] Reworked `ModelSubmissionHandler` API. This includes changes of method names and parameters [#119](https://github.com/eclipse-glsp/glsp-server/pull/96) [#197](https://github.com/eclipse-glsp/glsp-server/pull/101)
+- [operation] Refactored the `CreateNodeOperationHandler.createNode` method. The method now also passes the argument map of the operation. [#223](https://github.com/eclipse-glsp/glsp-server/pull/108)
+- [diagram] Refactored methods in`DiagramConfiguration` to correctly reflect `ShapeTypeHint` instead of `NodeTypeHint`.
+- [build] Update dependencies of Google Guava (>= 30.1) and Google Guice (>= 5.0.0). [#260](https://github.com/eclipse-glsp/glsp-server/pull/119)
+- [build] The `org.apache.log4j` dependency is not reexported any more when consumed as p2 artifact. You need to import `org.apache.log4j` in your own plugins if you use the log4j API there. [#430](https://github.com/eclipse-glsp/glsp/issues/430)
```

Comment:
This and a few other entries are present in the breaking changes and the regular changes section. I suggest to either only have changes in one of the two sections or to put ALL breaking changes in the regular changes section as well for consistency.